src/main/java/com/ezasm/gui/RegisterTable.java

```diff
@@ -111,6 +137,39 @@ public boolean isCellEditable(int row, int col) {
         public String getColumnName(int column) {
             return columns[column];
         }
+
     }
 
+    /**
+     * Tell the table what the changed resgiter when execute the line
+     *
+     * @param number the index of changed register
+     *
+     * @param config the instance used to get theme
+     */
+    public void addHighlightValue(Long number, Config config) {
+        if (!changedRegisterNums.contains(number.intValue())) {
+            this.changedRegisterNums.add(number.intValue());
+        }
+        this.config = config;
+    }
+
+    public void removeHighlightValue() {
+        this.changedRegisterNums.clear();
+    }
+
+    public void removeExtra() {
```

Comment:
remove extra what? Add javadoc to describe purpose, possibly rename this method as well.